support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -476,42 +473,43 @@ function PaymentRequestButton(props: PropTypes) {
 				return stripe.handleCardAction(clientSecret);
 			});
 		}
-
-		props.setPaymentRequestObject(paymentRequestObject);
```

Comment:
Before we were always setting the `PaymentRequest` object even if we couldn't make a payment with it. With the new state we only save it if it's available, else we save `{ status: 'NOT_AVAILABLE' }`